### PR TITLE
Docs software install

### PR DESCRIPTION
**Pull request by rayjh**

### CODE REVIEW

File: docs/Appendix-J.rst [hunk 1]
```diff
@@ -1,102 +1,147 @@
...
+===================================================================
 
-TODO: general description
+Overview
+--------
+The POWER-Up software can run on OpenPOWER or x_86 architecture.
 
-Save Container Files
---------------------
+The PowerAI Enterprise Software Install Module provides for rapid installation of the PowerAI Enterprise software to a cluster of Power8 or Power9 servers.
```
Comment (jaywcarman) [inline docs/Appendix-J.rst]: I'm not sure how official we need to be here, but I believe "POWER8" and "POWER9" should be written in all caps.

File: docs/Appendix-J.rst [hunk 2]
```diff
@@ -1,102 +1,147 @@
...
 
-#. From your home directory, clone Cluster Genesis::
+Install the POWER-Up software::
 
-    user@new-host:~$ git clone https://github.com/open-power-ref-design-toolkit/cluster-genesis
+    $ sudo yum install git
 
-#. Install the remaining software packages used by Cluster Genesis and
-   setup the environment::
+    $ git clone https://github.com/open-power-ref-design-toolkit/power-up -b software-install-b1
```
Comment (jaywcarman) [inline docs/Appendix-J.rst]: They'll also need to run `./scripts/install.sh` and `source ./scripts/setup-env`. We could link to the 'docs/Installing.html' page? (but then they might forget to checkout the 'software-install-b1' branch)

File: docs/Appendix-J.rst [hunk 3]
```diff
@@ -1,102 +1,147 @@
...
+
+**Dependent software packages**
+The PowerAI Enterprise software is dependent on additional open source software that is not shipped with PowerAI Enterprise.
+These dependent packages are downloaded to the POWER-Up software server from enabled yum repositories during the preparation phase and are subsequently available to the client nodes during the install phase.
+Additional software packages can be installed in the 'dependencies' repo on the POWER-Up software server by listing them in the power-up/software/dependent-packages.list file.
+Entries in this file can be delimited by spaces or commas and can appear on multiple lines.
+Note that packages listed in the dependent-packages.list file are not automatically installed on client nodes unless needed by the PowerAI software.
+They can be installed on a client node explicitly using yum on the client node (ie yum install pkg-name). Alternatively, they can be installed on all client nodes at once using Ansible::
+
+    $ ansible all -i software_hosts -m yum -a "name=pkg-name"
```
Comment (jaywcarman) [inline docs/Appendix-J.rst]: This command assumes the user will be in the 'playbooks/' directory (for the `-i software_hosts` path to work). I've been wondering if we should move it into the 'software/' directory with all of the other pai52 module code. In any case we should probably mention that this is a relative path dependent on the CWD.

File: docs/Setup.rst [hunk 4]
```diff
@@ -30,37 +33,34 @@ Operating Sytem and Package setup of the Deployer Node
 - Deployer OS Requirements:
     - Ubuntu (Software installation is not yet supported under Ubuntu)
         - Release 14.04LTS or 16.04LTS
-        - SSH login enabled
+        - SSH login enabled if the deployer node needs to be accessed remotely
```
Comment (jaywcarman) [inline docs/Setup.rst]: There are other ways a user could remote access a server so I don't think SSH (server) is _really_ a deployer requirement (/nickpick).

File: docs/Appendix-J.rst (hunk 1 above)
Comment (rayjh) [reply docs/Appendix-J.rst]: Yep

File: docs/Appendix-J.rst (hunk 2 above)
Comment (rayjh) [reply docs/Appendix-J.rst]: Good catch.  For the moment I'd like to keep this as simple to use as possible for the user.  They already have to jump from Knowledge center to here, so I'd like to minimize the jumping around here and put everything they need right in Appendix-J.    (The risk of course is that we make a mistake and leave something out :-) )

File: docs/Appendix-J.rst (hunk 3 above)
Comment (rayjh) [reply docs/Appendix-J.rst]: I think I like the idea of moving software_hosts to the software/ directory.

File: docs/Setup.rst (hunk 4 above)
Comment (rayjh) [reply docs/Setup.rst]: I'm going to remove this line altogether. We have to assume some level of competence on the part of the user.

**Pull request merged**